### PR TITLE
[#814] -Content and UX improvement after tests

### PR DESCRIPTION
**Pull request by abacz**
What's in the PR?
Fixes: #814

### CODE REVIEW

**Comment by kmarszalek** [comment]
remember to remove "sok.grid.cyfronet.pl" from urls in api howto :)

File: s4e-backend/seed-configs/seed-1/products.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
...
+    "description": " Kompozycja barwna RGB Natural Colors utworzona z kanałów spektralnych czujnika SEVIRI/Meteosat 1.6 µm, 0.8 µm i 0.6 µm. Pozwala na detekcję powierzchni lądu pokrytej wegetacją oraz rozróżnienia chmur lodowych od wodnych. Chmury lodowe, śnieg oraz lód na powierzchni widoczne są tu w cyjanie, przy czym odróżnienie śniegu na powierzchni gruntu od chmur lodowych jest możliwe jedynie na podstawie charakterystycznej struktury śniegu (o ile występuje). Z kolei fotosyntetycznie aktywna wegetacja zaznaczona jest kolorem zielonym (i jego odcieniami). Pustynie, odsłonięta gleba i wyschnięta wegetacja widoczne są w odcieniach brązu.",
```
Comment (jswk) [inline s4e-backend/seed-configs/seed-1/products.json:84]: This space is intentional?

**Review by jswk — APPROVED** [review]
Looks good for me

File: s4e-backend/seed-configs/seed-1/products.json (hunk 1 above)
Comment (abacz) [reply s4e-backend/seed-configs/seed-1/products.json:84]: Neh.

**Review by kmarszalek — APPROVED** [review]
waiting for changes in api howto in the next iteration

**Pull request merged**